### ISSUE TITLE
Signing Validation Error on bundle release

### ISSUE THREAD

**Issue by rupesh-kumar-lpu** [processing/processing-android#691]
![SigningErrorBundle](https://user-images.githubusercontent.com/46577873/167487875-0ba471e7-d084-49d8-b8e7-24303b944d4b.PNG)

It seems the Keystore path needs to be checked which is being used in the `gradle.properties` template. It looks like slashes are not being used in the returned Keystore file path.

I'm looking into it.

**Comment by codeanticode** [comment]
That's strange, I'm not getting this error when I build the signed aab on Mac. Have you tried running the task from the command line directly in the temporary build directory where Processing stores the Android project. The full path is printed in the console:

![image](https://user-images.githubusercontent.com/62246/173356260-29c472df-0265-4c1c-b352-eeb9cd5ca9ae.png)

**Comment by rupesh-kumar-lpu** [comment]
After a lot of debugging related to the signing path, I have found the reason for the issue. The issue seems to be platform-specific(occurs on windows). 
I tried running bundleRelease command directly at temp folder sketch path but the strange part was gradle.properties shows unchanged values as follows which creates a build error on direct execution-
```
org.gradle.jvmargs=-Xmx1536m
android.enableJetifier=true
android.useAndroidX=true

PROCESSING_UPLOAD_KEYSTORE_FILE=@@keystore_file@@
PROCESSING_UPLOAD_KEY_ALIAS=@@key_alias@@
PROCESSING_UPLOAD_STORE_PASSWORD=@@key_password@@
PROCESSING_UPLOAD_KEY_PASSWORD=@@key_password@@
```
So I assumed that the changes may reflect on runtime on processing4. So I printed gradle.properties file data on runtime and it was showing me the proper signing data as expected.
Later when I used `print storeFile` in ECJ gradle file, I found that it was not printing the correct value as mentioned in gradle.properties file and remove slashes as mentioned in the first screenshot on this issue.
Later, I changed the following line to include double slashes in the path of PROCESSING_UPLOAD_KEYSTORE_FILE in gradle.properties and the build was successful.
I changed this line-
[replaceMap.put("@@keystore_file@@", AndroidKeyStore.getKeyStore().getAbsolutePath());](https://github.com/processing/processing-android/blob/78b8bca79799b7d02a25112e4effc2a0ee3f43f6/mode/src/processing/mode/android/AndroidBuild.java#L387)
to the following-
`replaceMap.put("@@keystore_file@@", AndroidKeyStore.getKeyStore().getPath().replaceAll("\\\\", "\\\\\\\\"));`
which adds up the double slashes in the path of storeFile as follows-
`C:\\Users\\Name\\Documents\\Processing\\android\\keystore\\processing-upload-keystore.jks`

**In short**, if the path with a single slash is being used in gradle.properties for PROCESSING_UPLOAD_KEYSTORE_FILE
it shows a build error as mentioned in the top screenshot and `print storeFile` returns path without slashes.
If doubles back slashes are being used as mentioned by the above method of replace all, `print storeFile` returns path with single slashes, and the build gets successful.

I can resolve this with the condition to identify the platform and use `replaceMap.put` accordingly. But it would be better if I get confirmation from someone on running the Export Signed Bundle command on windows.

Thanks!

**Comment by rupesh-kumar-lpu** [comment]
@codeanticode thanks for clearing it Andres.
